html-css/week1/styles.css

```diff
@@ -0,0 +1,164 @@
+body{
```

Comment:
seems nice and organized, only thing I recommend is that you should use media queries for responsive adjustments, especially for .navbar-links, .skills, and .projects.